miso-web/src/main/webapp/scripts/list_note.js

```diff
@@ -0,0 +1,208 @@
+ListTarget.note = (function () {
+  return {
+    name: "Notes",
+    createUrl: function (config, entityId) {
+      var id = config.sampleId || config.libraryId || entityId;
+      var entityType = config.entityType;
+
+      if (!id || !entityType) {
+        console.error("Error: Missing entity ID or type for notes");
+        return null;
+      }
+
+      return "/rest/notes/" + entityType + "/" + id + "/dt?includeRelated=true";
+    },
+    getQueryUrl: function () {
+      return "/rest/notes/search";
```

Comment:
This is for the "Select by Names" feature, which doesn't make sense for this table. You can omit `getQueryUrl` entirely